### ISSUE TITLE
Automatic crash

### ISSUE THREAD

**Issue by sanyi12** [effekseer/EffekseerForGodot3#11]
The Godot version 3.4 crashes automatically when trying to add a emitter node to scene. The version 3.4.2 doesn't crash but the emitter node is not working. Can someone tell me which version works?

**Comment by sanyi12** [comment]
I also tried the live version of the plugin and still doesn't work

**Comment by ueshita** [comment]
Please tell us your OS environment?

**Comment by sanyi12** [comment]
Windows 10

**Comment by ueshita** [comment]
OK.
and we have released a new version.
https://github.com/effekseer/EffekseerForGodot3/releases/tag/162b
Do you still have issues using this?

**Comment by unicornlox** [comment]
without adding camera node, not adding effekseeremitter node.The spatial node must be added first, and then the camera node.Finally, while the spatial node is active, the effekseeremitter node must be added.

**Comment by game009f** [comment]
![2022-03-17-13-52-39](https://user-images.githubusercontent.com/80683955/158745688-01389880-acb8-411a-8520-3510d3544e83.png)
The node of "effekseemitter (effekseemitter. GNDS)" must be selected to work normally. Select effekseemitter and the engine crashes

**Comment by sanyi12** [comment]
Hi, I reinstalled my windows and installed the newest Godot (3.4.4) and the newest version of this addon and still when trying to add a 3d emitter it crashes the engine. Could you perhaps make an addon to blender instead? exporting from blender to Godot would be easier then. Also it will help with code maintainability. You work on UNITY, UNREAL and GODOT exporters right now. It's crazy. Make 1 good exporter for Blender and maintain that. Everyone is using Blender nowadays.

**Comment by thalesrca** [comment]
> without adding camera node, not adding effekseeremitter node.The spatial node must be added first, and then the camera node.Finally, while the spatial node is active, the effekseeremitter node must be added.

Exactly. I was facing the same problem in some scenes (crashing godot), so I had to add a camera first and only then add the effekseer node. Maybe the camera it is a dependency or something like that. If it is the case, I think it would be nice to have a message explaining instead of crashing the engine
